elasticdl/python/master/master.py

```diff
@@ -46,20 +46,7 @@ def prepare(self):
                 self.pod_manager._remove_worker
             )
         if self.pod_manager:
-            if self.elasticdl_job_service:
-                command = self.elasticdl_job_service.get_ps_worker_command()
-                self.pod_manager.set_up(
-                    worker_command=command,
-                    worker_args=self.elasticdl_job_service.get_worker_args(
-                        self._args
-                    ),
-                    ps_command=command,
-                    ps_args=self.elasticdl_job_service.get_ps_args(self._args),
-                )
-            else:
-                # TODO: Get the Pod arguments from the input
-                # args directly
-                pass
+            self._set_pod_manager()
```

Comment:
`_set_pod_manager` name does not show what this function is doing.
`set_command_in_pod_manager` ?

Reply:
Done!